File: week-02/day-02/summing.py
# - Write a function called `sum` that sum all the numbers
#   until the given parameter
input_number = int(input("Please input a number "))
def sum_(number):
    if number > 0:
        return sum(list(range(1,number+1)))
    else:
        return sum(list(range(number,0)))
print(sum_(input_number))

File: week-07/day-03/summing.js
'use strict';
// - Write a function called `sum` that sum all the numbers until the given parameter
// - The function should return the result
function summing(number) {
    let summa = 0
    for (let i = 1; i < number+1; ++i) {
        summa += i
    }    
    return summa
}

console.log(summing(6));